pubber/src/main/java/daq/pubber/Pubber.java

```diff
@@ -152,13 +154,14 @@ DiscoveryEvent.class, getEventsSuffix("discovery")
   private final PubberConfiguration configuration;
   private final AtomicInteger messageDelayMs = new AtomicInteger(DEFAULT_REPORT_SEC * 1000);
   private final CountDownLatch configLatch = new CountDownLatch(1);
-  private final State deviceState = new State();
+  final State deviceState = new State();
   private final ExtraPointsetEvent devicePoints = new ExtraPointsetEvent();
   private final Set<AbstractPoint> allPoints = new HashSet<>();
   private final AtomicBoolean stateDirty = new AtomicBoolean();
   private final Semaphore stateLock = new Semaphore(1);
   private final String deviceId;
-  private Config deviceConfig = new Config();
+  private final List<Entry> logentries = new ArrayList<>();
```

Comment:
logEntries?

Reply:
logentries is congruent with the use (where the full string is `deviceConfig.system.logentries` or something like that) -- I suppose it doesn't really matter, but I was just going with consistency at that level.